### PR TITLE
Calendar Fetching

### PR DESCRIPTION
**Pull request by tejeshwar-nt**

### CODE REVIEW

**Comment by tejeshwar-nt** [comment]
https://github.com/user-attachments/assets/85e20a0d-22f1-4701-a442-b30441d9f301

File: CanvasPlusPlayground.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -443,7 +451,7 @@
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
-				PRODUCT_BUNDLE_IDENTIFIER = com.rahulon.CanvasPlusPlayground;
+				PRODUCT_BUNDLE_IDENTIFIER = "com.jiyoon-wd.CanvasPlusPlayground";
```
Comment (rahulon12) [inline CanvasPlusPlayground.xcodeproj/project.pbxproj:454]: Could you unstage these changes?

File: CanvasPlusPlayground/CalendarManager.swift
Comment (rahulon12) [inline CanvasPlusPlayground/CalendarManager.swift:1]: I don't think this file has been added to your `project.pbxproj` file. When I pulled it doesn't show up in Xcode.

File: CanvasPlusPlayground/Views/CalendarView.swift [hunk 2]
```diff
@@ -8,12 +8,136 @@
...
+                }
+            }
+        } else {
+            Text("No calendar available")
+                .navigationTitle("Calendar")
+        }
+    }
+}
+
+struct EventSection: View {
```
Comment (rahulon12) [inline CanvasPlusPlayground/Views/CalendarView.swift:49]: nit: 
```suggestion
private struct EventSection: View {
```

File: CanvasPlusPlayground/Views/CalendarView.swift [hunk 3]
```diff
@@ -8,12 +8,136 @@
...
+    var body: some View {
+        Section(header: Text(date?.toDayMonthDay() ?? "No Date")) {
+            ForEach(events) { event in
+                EventDetails(event: event)
+            }
+        }
+    }
+}
+
+struct EventDetails: View {
```
Comment (rahulon12) [inline CanvasPlusPlayground/Views/CalendarView.swift:62]: nit:
```suggestion
private struct EventDetails: View {
```

**Review by rahulon12 — CHANGES_REQUESTED** [review]
We need to figure out why the date isn't appearing correctly, like we discussed in person.

**Pull request closed without merging**